backend/tracim_backend/lib/core/content.py

```diff
@@ -324,7 +324,7 @@ def _revisions_base_query(self, workspace: Workspace = None) -> Query:
     def get_base_query(self, workspace: typing.Optional[Workspace]) -> Query:
         return self._base_query(workspace)
 
-    # TODO - G.M - 2018-07-17 - [Cleanup] Drop this method if unneeded
+    # TODO - G.M - 2020-09-29 - [Cleanup] Should probably be dropped, see issue #704
     # def get_child_folders(self, parent: Content=None, workspace: Workspace=None, filter_by_allowed_content_types: list=[], removed_item_ids: list=[], allowed_node_types=None) -> typing.List[Content]:
     #     """
     #     This method returns child items (folders or items) for left bar treeview.
```

Comment:
Let's just drop this code commented since 2018, what do you think?

Reply:
i do prefer we clean all at the same time see #704

Reply:
remove it in another non-related PR make thing a bit more complicated to follow.